cdxev/__main__.py

```diff
@@ -491,6 +503,81 @@ def create_merge_vex_parser(
     return parser
 
 
+# noinspection PyUnresolvedReferences,PyProtectedMember
+def create_vex_parser(
+    subparser: argparse._SubParsersAction,
+) -> argparse.ArgumentParser:
+    parser = subparser.add_parser(
+        "vex",
+        help="Executes commands [list, search, trim, extract] on VEX/SBOM files.",
+    )
+
+    subparsers = parser.add_subparsers(dest="sub_command", required=True)
+
+    list_parser = subparsers.add_parser(
+        "list", help="Returns a list of all vulnerability IDs."
+    )
+    list_parser.add_argument(
+        "--scheme",
+        help="Set scheme of return list.",
+        choices=["default", "lightweight"],
+        default="default",
+        type=str,
+    )
+
+    list_parser.add_argument(
+        "--format",
+        help="Set format of return file.",
+        choices=["txt", "csv"],
+        default="csv",
+        type=str,
+    )
+
+    search_parser = subparsers.add_parser("search", help="Get vulnerabilities by ID.")
+    search_parser.add_argument(
+        "vul_id",
+        metavar="<vul_id>",
+        help="The ID of the vulnerability to search for.",
+        type=str,
+    )
+
+    trim_parser = subparsers.add_parser(
+        "trim",
+        help="Trims a VEX to show only the vulnerabilities according to the selected state.",
+    )
+    trim_parser.add_argument(
+        "--state",
+        help="Specifies the state to be filtered.",
+        choices=[
+            "resolved",
+            "resolved_with_pedigree",
+            "exploitable",
+            "in_triage",
+            "false_positive",
+            "not_affected",
+        ],
```

Comment:
I would refer from using choices for the state. If the states change sometimes e.g. due to specification changes, you have to differentiate between versions. Something you really should avoid. Why not just compare the value with the value within `state`? If it does not equal anything, you simply can ignore the entry. I do understand that you want to help the user, but somebody working with the VEX here, will have the documentation in mind.